### DISCUSSION TITLE
unable to compile raylib and miniaudio

### DISCUSSION THREAD

**Discussion by Lucas-Marcel** [raysan5/raylib#3192] [Q&A]
I'm getting many errors whenever i try to compile a sample code with raylib and miniaudio, the errors occur just by adding #define MINIAUDIO_IMPLEMENTATION, If i disable this line of code it compiles with no issue, but it doesn't work as intended. Here are the errors https://pastebin.com/ybJXpC4n

**Comment by raysan5** [comment]
raylib already includes miniaudio linked by default, so no need to define `MINIAUDIO_IMPLEMENTATION` again.

**Reply by Lucas-Marcel** [reply]
Yeah that helped. I managed to make it work. Using raylibs bundle with notepad++ and mingw still gives me errors. So i decided to build raylib from source code and it finally worked. But now im having issues trying to build it with tiny c and im running into similar errors. While i can build and run miniaudio and libxmp with tiny c compiler, the same cant be done if i add raylib the project. So i can only do that with mingw.
